assets/src/stories-editor/components/resizable-box/test/getPositionAfterResizing.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import { getPositionAfterResizing } from '../helpers';
+
+describe( 'getPositionAfterResizing', () => {
+	it( 'should return correct left and top position for text block', () => {
```

Comment:
As the issues with #3444 . Can we also add tests to resize from top and right would prove that #3444 was fixed.

Reply:
There are some resizing tests which are resizing from top which are [failing right now](https://travis-ci.org/ampproject/amp-wp/jobs/598763242#L920-L970), I'm looking into those but it's possible that these are actually failing now due to #3444 being fixed, perhaps these were incorrect before.

Reply:
After some investigation came to the conclusion that this was indeed the case: the tests had incorrect results since it jumped before when resizing. Adjusted the values within fe5cba0.